google/cloud/forseti/services/inventory/storage_helpers.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2018 The Forseti Security Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Helper functions for data transformation."""
+
+from google.cloud.forseti.services.inventory.storage import DataAccess
+from google.cloud.forseti.services.inventory.storage import Storage
+
+
+def get_user_emails(service_config, member_types=None):
+    """Retrieves the list of user email addresses from inventory.
+
+    Args:
+        service_config (dict): The service configuration
+        member_types (list): Member types to query in storage. This
+            defaults to 'gsuite_user'.
+
+    Returns:
+        list: List of list of user e-mail addresses.
+    """
+    if not member_types:
+        member_types = ['gsuite_user']
+
+    emails = []
+    with service_config.scoped_session() as session:
+        inventory_index_id = \
```

Comment:
Please don't use \ for line continuation, or style guide states to use () to wrap a multiline wrap.

For example:

inventory_index_id = (
    DataAccess.get_latest_inventory_index_id(session))